ci/build_wheel_libcudf.sh

```diff
@@ -36,7 +36,7 @@ export SKBUILD_CMAKE_ARGS="-DUSE_NVCOMP_FROM_LIBKVIKIO_WHEEL=ON"
 
 # repair wheels and write to the location that artifact-uploading code expects to find them
 python -m auditwheel repair \
-    --exclude libnvcomp.so.4 \
+    --exclude libnvcomp.so.* \
```

Comment:
@vuule Aha. This excludes nvcomp 5 from the cudf wheel. Previously we were safe because the kvikio wheel was shipping nvcomp libraries for us. We need to update kvikio first, or just delete this line for testing.

Reply:
Let's land https://github.com/rapidsai/kvikio/pull/798 first and then come back to this.